tests/testthat/_snaps/centralization.md

```diff
@@ -0,0 +1,27 @@
+# centr_eigen_tmax() deprecated argument
+
+    Code
+      c <- centr_eigen_tmax(g, scale = FALSE)
+    Condition
+      Warning:
+      The `scale` argument of `centr_eigen_tmax()` is deprecated as of igraph 2.1.3.
+      i The function always behaves as if `scale` were TRUE. The argument will be removed in the future.
+
+# centr_eigen() deprecated argument
+
+    Code
+      c <- centr_eigen(g, scale = FALSE)
+    Condition
+      Warning:
+      The `scale` argument of `centr_eigen()` is deprecated as of igraph 2.1.3.
+      i The function always behaves as if `scale` were TRUE. The argument will be removed in the future.
+
+# centr_degree_tmax() deprecated argument
+
+    Code
+      c <- centr_degree_tmax(g)
+    Condition
+      Warning:
+      The `loops` argument of `centr_degree_tmax()` must be explicit as of igraph 2.0.0.
+      i Default value (`FALSE`) will be dropped in next release, add an explicit value for the loops argument.
```

Comment:
I actually wonder why an explicit value is needed here.

Reply:
There is no reasonable default. The user should be forced to make a decision. The decision can only be made based on an understanding of where the network came from—it can't be deduced by looking at the network structure.

Reply:
Ooooh I understand now. I think the current message is slightly misleading: I thought support for TRUE was getting dropped, not support for not choosing. :sweat_smile:

Reply:
But now that I know what it means I am not sure how to rephrase the message.

Reply:
Hm, to me it seems clear (?), it just needs some articles: "The default values", "in the next release". Is it the mention of what the default was (i.e. `FALSE`) that you find confusing?